.github/workflows/api-unit-tests.yml

```
@@ -0,0 +1,40 @@
+      - name: Install dependencies
+        run: bun install
+      
+      - name: Run unit tests
+        run: bun run test --exclude "**/__tests__/**"
+        env:
+          DATABASE_URL: "postgresql://test:test@localhost:5432/test"
+          IPFS_KEY: "test-key"
+          IPFS_GATEWAY_WRITE: "https://ipfs.io"
+          RPC_ENDPOINT: "https://rpc.testnet.example.com"
+          CHAIN_ID: "80451"
+          DEPLOYER_PK: "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)